docs/docs/partials/_InitializeDgProject.md

```diff
@@ -0,0 +1,30 @@
+import InstallUv from '@site/docs/partials/\_InstallUv.md';
+
+<Tabs groupId="package-manager">
+    <TabItem value="uv" label="uv">
+        Ensure you have `dg` [installed globally](/guides/labs/dg) as a `uv` tool:
+
+        <CliInvocationExample contents="brew install uv && uv tool install dagster-dg" />
```

Comment:
this is too mac-specific. 

I would have each of these in a separate cli invocation and make link to uv for installation instructions.

Reply:
also can we nest tabs? that would be ideal

Reply:
I added tabbed uv installation instructions